### PR TITLE
Python

### PR DESCRIPTION
**Pull request by icimrak**
Fixes #

Description of changes:
 - 

Added particle feature mol to python interface, added one line to testsuite/python/particel.py to test the mol-id

PR Checklist
------------
 - [ ] Tests?
   - [ ] Interface
   - [ ] Core 
 - [ ] Docs?

### CODE REVIEW

**Comment by hmenke** [comment]
Your PR has merge conflicts.  Please rebase.

File: src/python/espressomd/reaction.pyx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from __future__ import print_function, absolute_import
 include "myconfig.pxi"
+cimport globals
```
Comment (hmenke) [inline src/python/espressomd/reaction.pyx]: This *has* to be
````python
from . cimport globals
````
for Python 3 compatibility.

**Comment by RudolfWeeber** [comment]
This pr seems to contain more commits than the one mentioned in the description.
Was that intended?
If not, please take the current master and just 
git cherry-pick
the "mol" commit into it.

**Comment by icimrak** [comment]
After advice of Rudolf, I will close this pull request and implement mol feature from clean espressomd/python branch

Ivan

**Comment by icimrak** [comment]
Hi Rudolf,


It was indeed not intended. So I closed the previous pull request, got 
the fresh branch espressomd/python and implemented the mol feature 
there. Then I create new pull request, awaiting the response....


Thanks,

Ivan

**Pull request closed without merging**